app/ts/components/TokenManager/index.tsx

```diff
@@ -113,7 +113,9 @@ const TokenCard = ({ token, onSelect }: TokenCardProps) => {
 	return (
 		<div class='aspect-square bg-neutral-800 p-4 flex items-end relative group outline-none' tabIndex={1}>
 			<div class='relative'>
-				<img class='w-12 h-12 mb-2' src={`/img/${token.address.toLowerCase()}.svg`} />
+				<object class='w-12 h-12 mb-2' data={`/img/${token.address.toLowerCase()}.svg`} type="image/png">
+					<img class='w-12 h-12' src="/img/ethereum.svg" />
```

Comment:
We shouldn't use the ETH logo as a fallback for missing logo.  Consider a question mark or some other indicator of "unknown" to clearly indicate that we don't have an icon.

Reply:
Replaced the icon with this:
<img width="69" alt="Screenshot 2023-07-06 at 3 02 48 PM" src="https://github.com/DarkFlorist/lunaria/assets/1169838/a295d565-5c85-42e8-b141-97c920de032f">

Reply:
Hmm, screenshot missing?